backend/src/game/dice_handler.py

```diff
@@ -0,0 +1,26 @@
+import backend.src.game.dice as dice
+import backend.src.game.constants as constants
+
+
+class DiceHandler:
+    dice_values = []
```

Comment:
Class variables although are able to be defined as non-static, should typically be static.  Also this would be getting shared across instances and I am unsure how you are planning on using the dice handler but I do not think that is the best way of storing this

Reply:
I'm trying to follow up on this one to make sure I understand but having trouble finding exactly what you're referencing.  Generally If I'm understanding correctly, I'm finding static variables are not really a thing in python like other languages. Do you mean the difference between class and instance variables? I found this, https://syntaxdb.com/ref/python/class-variables and realized I didn't understand the difference between putting things in the __init__ method and where I put it.

Reply:
I moved the values to be instance values instead of class variables. I think that addresses your concerns